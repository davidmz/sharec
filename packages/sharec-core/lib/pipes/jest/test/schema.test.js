const { fixtures } = require('testUtils')
const { jestJson } = require('../schema')

describe('pipes > jest > schema', () => {
  describe('JSON', () => {
    const jestBaseFxt = fixtures('jest/json/00-base', 'map')

    it('should merge configs', () => {
      expect(jestJson(jestBaseFxt)).toMatchSnapshot()
    })
  })
})
