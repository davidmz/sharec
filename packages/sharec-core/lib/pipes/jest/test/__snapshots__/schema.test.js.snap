// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pipes > jest > schema JSON should merge configs 1`] = `
Map {
  "testEnvironment" => "node",
  "clearMocks" => true,
  "cacheDirectory" => "./.cache",
  "extraGlobals" => Array [
    "foo",
    "bar",
    "baz",
  ],
  "coverageThreshold" => Map {
    "global" => Map {
      "statements" => 100,
    },
  },
}
`;
